src/cli/__tests__/run-cli.ts

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
```

Comment:
Did you try to use shebang? something like `#!./node_modules/.bin/ts-node`. Or `ts-node/register` module? Maybe it'll be easier than this.

Reply:
You mean to add `#!./node_modules/.bin/ts-node` do `cli/index.ts`? In this case after ts compilation js file will include wrong shebang.
Using this file I can test setting option via command line interface.

Reply:
Oh my, I thought this file is used to run the program by the user. Didn’t notice that it’s inside ‘tests’ directory. Sorry, shame on me!

Reply:
One problem with this solution I can't mock main script, so it will e2e test. Is it okey?

Reply:
That’s perfectly right. Otherwise you’d test commander configuration only, what’s even not worth of the effort I think.